src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test/MessageInterception/DefaultInterceptionManagerTest.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.ContainedLanguage.MessageInterception;
+using Moq;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+#nullable enable
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.MessageInterception
+{
+    public class DefaultInterceptionManagerTest
+    {
+        [Fact]
+        public void Ctor_NullArguments_Throws()
+        {
+            Assert.Throws<ArgumentNullException>(() => new DefaultInterceptorManager(null!));
```

Comment:
Hahaha, I see you took my "we test every call path" pretty seriously. For future reference we don't typically test the default `null` checking that we add but there's no harm in having it here 😄

Reply:
I think it's valuable to have these tests for two reasons:
- the unit testing makes the contract explicit about what happens with null values
- for a mixed-state world of #nullable annotations, this verifies that you've considered a non-nullable aware consumer of your code.  Your comment above about adding better null checks was exactly on-point with this.